Content.Server/CartridgeLoader/Cartridges/CrewManifestCartridgeSystem.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Content.Server.CrewManifest;
+using Content.Server.Station.Systems;
+using Content.Shared.CartridgeLoader;
+using Content.Shared.CartridgeLoader.Cartridges;
+using Content.Shared.CCVar;
+using Robust.Shared.Configuration;
+
+namespace Content.Server.CartridgeLoader.Cartridges;
+
+public sealed class CrewManifestCartridgeSystem : EntitySystem
+{
+
+    [Dependency] private readonly CartridgeLoaderSystem? _cartridgeLoader = default!;
```

Comment:
This dependency can't ever be nullable so no need, same with below.

Reply:
Fixed